collector/server.py

```diff
@@ -55,27 +55,28 @@ def __init__(self, metric_name, metric_value, is_error=False):
         self.metric_value = metric_value
         self.is_error = is_error
 
-def get_snmp_data(ip):
+def get_snmp_data(ips):
```

Comment:
can we do it like below, then you dont need to change `get_snmp_data` at all

```py
def work(ip_list):
    while True:
        for ip in ip_list:
            get_snmp_data(ip)
        time.sleep(args.sleep_duration_minutes * 60)
```

Reply:
done